config/toml.go

```diff
@@ -70,7 +70,7 @@ block_time = "{{ .BlockManagerConfig.BlockTime }}"
 # block production interval in case of no transactions ("0s" produces empty blocks)
 max_idle_time = "{{ .BlockManagerConfig.MaxIdleTime }}"
 max_proof_time = "{{ .BlockManagerConfig.MaxProofTime }}"
-max_supported_batch_skew = {{ .BlockManagerConfig.MaxBatchSkew }}
+max_supported_block_skew = {{ .BlockManagerConfig.MaxBlockSkew }}
```

Comment:
why is it called `max_supported_block_skew` (which is a system definition) if its a user defined param (i.e `max_block_skew`)

Reply:
renamed